src/app.module.ts

```diff
@@ -23,8 +18,6 @@ import { ConfigurationModule } from './common/config/configuration.module';
 })
 export class AppModule implements NestModule {
   configure(consumer: MiddlewareConsumer) {
-    consumer
-      .apply(LoggerMiddleware)
-      .forRoutes({ path: '*', method: RequestMethod.ALL });
```

Comment:
Was this done automatically? I actually have a slight preference for the left part since it looks like a builder pattern.

Reply:
Yes, almost all changes were done automatically, I'm ok with both versions, but I can preserve the old one if you prefer 👍

Reply:
I was thinking we could preserve the left part if the line length is not reached but it seems prettier enforces styling even if the line length is not reached. At this point we have automatic formatting when committing, so I would take one of the options: 1. close this PR and stay on the 80 chars limit. 2. assume that prettier will format this.

What do you think @fmrsabino? Do you see other options here? 🙂